scrunch/datasets.py

```diff
@@ -1768,103 +1285,14 @@ def savepoint_attributes(self, attrib):
                 'user_name'
                 'version'
         """
-
-        if len(self.resource.savepoints.index) == 0:
-            return []
-        else:
+        if len(self.resource.savepoints.index) != 0:
```

Comment:
Factor out this `self.resource.savepoints` so when you use it again below you don't have to do the GET twice